php/libraries/Candidate.class.inc

```diff
@@ -681,6 +681,33 @@ class Candidate
         return $vLabel;
     }
 
+    /**
+     * Returns list of consents and value of each for this candidate
+     *
+     * @param int $candID Candidate for which consent details are required
+     *
+     * @return array List of consents and their associated values for this candidate.
+     *               The keys of the arrays are the IDs of the consents
+     */
+    function getConsent($candID)
+    {
+        $factory = NDB_Factory::singleton();
+        $db      = $factory->database();
+
+        $query       = "SELECT *
```

Comment:
Please specify the columns. 
1. They are no all used 
2. If some get changed in the schema, it is not obvious that it was used here? 
3. If this need to join other tables, it will gather all those new table's columns

Reply:
agree